games-emulation/rmg/files/rmg-0.6.6-mupen64plus-input-raphnetraw-pkgconfig.patch

```diff
@@ -0,0 +1,46 @@
+https://bugs.gentoo.org/941889
+https://github.com/raphnet/mupen64plus-input-raphnetraw/pull/17
+https://github.com/Rosalie241/RMG/pull/296
+https://github.com/Rosalie241/RMG/commit/f6b73b86c6625fa5a268b74b1e36114a5bdd535c
+
+From f6b73b86c6625fa5a268b74b1e36114a5bdd535c Mon Sep 17 00:00:00 2001
+From: orbea <orbea@riseup.net>
+Date: Sun, 20 Oct 2024 08:59:56 -0700
+Subject: [PATCH] 3rdParty: don't hard-code pkg-config in
+ mupen64plus-input-raphnetraw (#296)
+
+---
+ .../projects/unix/Makefile                           | 12 ++++++++++--
+ 1 file changed, 10 insertions(+), 2 deletions(-)
+
+diff --git a/Source/3rdParty/mupen64plus-input-raphnetraw/projects/unix/Makefile b/Source/3rdParty/mupen64plus-input-raphnetraw/projects/unix/Makefile
+index a2571378..34430e6a 100644
+--- a/Source/3rdParty/mupen64plus-input-raphnetraw/projects/unix/Makefile
++++ b/Source/3rdParty/mupen64plus-input-raphnetraw/projects/unix/Makefile
+@@ -141,6 +141,14 @@ ifeq ($(OS), OSX)
+   endif
+ endif
+ 
++# test for essential build dependencies
++ifeq ($(origin PKG_CONFIG), undefined)
++  PKG_CONFIG = $(CROSS_COMPILE)pkg-config
++  ifeq ($(shell which $(PKG_CONFIG) 2>/dev/null),)
```

Comment:
I reviewed this in the (merged) upstream PR. It is not a great solution -- in order to use this patch, you'd have to add a new BDEPEND="sys-apps/which"

Reply:
I was copying the check from elsewhere in the project, but I believe because `PKG_CONFIG` is set in the ebuild this code will never be reached by Gentoo.

Reply:
While this is true for the above code I neglected the part right below it so I revised the patch upstream to avoid `which(1)` in mupen64plus-input-raphnetraw. However there appear to be other uses of `which(1)` in mupen64plus-core which I didn't notice before so I added the `sys-apps/which` dependency regardless. I will see about trying to remove it upstream later.

Reply:
Ok thanks. I don't love depending on it at all but it's hardly the only package to do it. The most important thing at the moment is actually just to get a good model of what packages actually have to have it installed as part of https://bugs.gentoo.org/show_bug.cgi?id=which-hunt

Reply:
I agree that it would be better to not depend on it and made this PR for mupen64plus-core that would avoid it in Gentoo, but I will have to see what they think.

https://github.com/mupen64plus/mupen64plus-core/pull/1093

Reply:
I updated this PR so that RMG no longer depends on `which` when `PKG_CONFIG` is explicitly set by the ebuild.

I also added the missing dependency to `0.6.5`.